.env.example

```diff
@@ -25,6 +25,8 @@ AUTH0_DOMAIN=your_Auth0_domain
 # Stripe
 STRIPE_SECRET_KEY=sk_test_51IravfFqipIA40A3FOW6EzlXlJiXjL9V0FXKfb9n7cxh25Ww9QMA9aWwCzTSQscBOQFcB7s1TI6UCtW1JG83Hz1z000Sg2vSIr
 
+#twilio
+
```

Comment:
Are we using a Twilio API key here? If so, we should have a blank environment variable assignment here for reference. If not, having the comment doesn't seem necessary.